mysql-test/suite/innodb/t/encryption_force.test

```diff
@@ -0,0 +1,107 @@
+-- source include/have_innodb.inc
+-- source include/not_embedded.inc
+-- source include/have_partition.inc
+
+--let $keyring2_restart_param=restart: --early-plugin-load="keyring_file=$KEYRING_PLUGIN" --loose-keyring_file_data=$MYSQL_TMP_DIR/mysecret_keyring2 $KEYRING_PLUGIN_OPT
+
+call mtr.add_suppression("\\[Error\\] InnoDB: Can't generate new master key for tablespace encryption, please check the keyring plugin is loaded.");
+call mtr.add_suppression("\\[Error\\] InnoDB: Encryption can't find master key, please check the keyring plugin is loaded.");
+call mtr.add_suppression("\\[ERROR\\] Function 'keyring_file' already exists");
+call mtr.add_suppression("\\[ERROR\\] Couldn't load plugin named 'keyring_file' with soname 'keyring_file.*'.");
+call mtr.add_suppression("Plugin keyring_file reported");
+
+# Restart the server with keyring loaded
+--let $restart_parameters="$keyring2_restart_param"
+--let $restart_hide_args= 1
+--source include/restart_mysqld.inc
```

Comment:
This is the sole restart in the testcase - can it be performed by the testcase options file?